.jshintrc

```diff
@@ -1,6 +1,6 @@
 {
     "node": true,
-    "esversion": 6,
+    "esnext": true,
```

Comment:
I changed this back since I couldn't find a version of the Atom JSHint that supports it. Once this is rolled out everywhere we should just create a script to update it for all modules at once.